### ISSUE TITLE
auth failing

### ISSUE THREAD

**Issue by tsvi** [home-assistant/supervisor#1694]
**Home Assistant release with the issue:**

```
arch | aarch64
-- | --
dev | false
docker | true
hassio | true
os_name | Linux
os_version | 4.19.114-v8
python_version | 3.7.7
timezone | Asia/Jerusalem
version | 0.109.2
virtualenv | false
```

**Operating environment (HassOS/Generic):**

**Supervisor logs:**

**Description of problem:**

1. In the system log I get the following error once every 5 minutes:

`20-05-03 09:23:39 ERROR (MainThread) [supervisor.auth] Can't request auth on Home Assistant!`

2. Every time I login I get the following notification:

Login attempt or request with invalid authentication from 172.30.32.2 (that's the reverse proxy

I have a reverse proxy setup with Nginx Proxy Manager and have the following in my config:

```
http:
  base_url: !secret hass_uri
  use_x_forwarded_for: true
  trusted_proxies:
  - 172.16.0.0/24
```

**Comment by frenck** [comment]
`172.16.0.0/24` does not include `172.30.32.2`... So the source IP is not visible...

**Comment by tsvi** [comment]
🤦‍♂️ changed it to 172.16.0.0/12 (the private up range), but still failing.

**Comment by frenck** [comment]
Well, the real source IP should now be revealed...?

**Comment by tsvi** [comment]
It looks according to the timing of the error that it is the supervisor that's trying to access it (error 1 I mentioned in the issue). How is the supervisor supposed to access home assisstant? Is it supposed to have a token that maybe is missing/was deleted? Or maybe I'm supposed to whitelist its IP?

**Comment by tsvi** [comment]
With the latest release of the supervisor, I managed to discover the culprit as the log now had the following message as well:

```20-05-10 05:25:55 INFO (MainThread) [supervisor.auth] Auth request from core_mosquitto for my_user```

Took me a few minutes, as MQTT seemed to work fine and my camera which I hooked up via MQTT was passing the data fine. Until o figured it was the zigbee2mqtt add-on I installed, started but never configured.

Once I setup username and password for the core_mosquitto broker, all is fine now.

Thanks,

Tsvi
